### DISCUSSION TITLE
HELP - Customizing

### DISCUSSION THREAD

**Discussion by RJaikanth** [mmistakes/minimal-mistakes#2879] [Q&A]
How do I customize the theme for github pages? I need to make my avatar photo square.
I have forked the starter repo and working from there.

**Comment by kulbhushanchand** [comment]
Add the following to the `custom.scss`, to override the theme default setting for the sidebar image.

```
img {
    max-width: 200px;
    border-radius: 0%;

    @include breakpoint($large) {
      padding: 0px;
      border: 0px solid $border-color;
    }
  }
```
In case you want to make further change to the theme, [this guide](https://www.cross-validated.com/Personal-website-with-Minimal-Mistakes-Jekyll-Theme-HOWTO-Part-II/) will be helpful.
